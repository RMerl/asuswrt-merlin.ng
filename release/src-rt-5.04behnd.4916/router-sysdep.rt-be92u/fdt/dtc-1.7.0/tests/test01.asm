/* autogenerated by dtc, do not edit */

#define OF_DT_HEADER 0xd00dfeed
#define OF_DT_BEGIN_NODE 0x1
#define OF_DT_END_NODE 0x2
#define OF_DT_PROP 0x3
#define OF_DT_END 0x9

	.globl	dt_blob_start
dt_blob_start:
_dt_blob_start:
	.globl	dt_header
dt_header:
_dt_header:
	.long	OF_DT_HEADER /* magic */
	.long	_dt_blob_end - _dt_blob_start /* totalsize */
	.long	_dt_struct_start - _dt_blob_start /* off_dt_struct */
	.long	_dt_strings_start - _dt_blob_start /* off_dt_strings */
	.long	_dt_reserve_map - _dt_blob_start /* off_dt_strings */
	.long	16 /* version */
	.long	16 /* last_comp_version */
	.long	0	/*boot_cpuid_phys*/
	.long	_dt_strings_end - _dt_strings_start	/* size_dt_strings */
	.balign	8
	.globl	dt_reserve_map
dt_reserve_map:
_dt_reserve_map:
/* Memory reserve map from source file */
	.long	0x10000000
	.long	0x00000000
	.long	0x00000000
	.long	0x02000000
	.long	0x20000000
	.long	0x00000000
	.long	0x01000000
	.long	0x00000000
	.long	0x00000000
	.long	0x00000000
	.long	0x00000000
	.long	0x00000014
	.long	0, 0
	.long	0, 0
	.globl	dt_struct_start
dt_struct_start:
_dt_struct_start:
	.long	OF_DT_BEGIN_NODE
	.asciz	""
	.balign	4
	.long	OF_DT_PROP
	.long	0xc
	.long	0x0
	.long	0x4d79426f
	.long	0x6172644e
	.long	0x616d6500
	.balign	4
	.long	OF_DT_PROP
	.long	0x1e
	.long	0x6
	.long	0x4d79426f
	.long	0x6172644e
	.long	0x616d6500
	.long	0x4d79426f
	.long	0x61726446
	.long	0x616d696c
	.long	0x794e616d
	.short	0x6500
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x11
	.long	0x2
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x20
	.long	0x2
	.balign	4
	.long	OF_DT_BEGIN_NODE
	.asciz	"cpus"
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x2c
	.long	0x1
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x11
	.long	0x1
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x20
	.long	0x0
	.balign	4
	.long	OF_DT_BEGIN_NODE
	.asciz	"PowerPC,970@0"
	.balign	4
	.long	OF_DT_PROP
	.long	0xc
	.long	0x3a
	.long	0x506f7765
	.long	0x7250432c
	.long	0x39373000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x3f
	.long	0x63707500
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x4b
	.long	0x0
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x4f
	.long	0x5f5e1000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x5f
	.long	0x1fca055
	.balign	4
	.long	OF_DT_PROP
	.long	0x0
	.long	0x72
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x81
	.long	0x10000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x8e
	.long	0x8000
	.balign	4
	.long	OF_DT_END_NODE
	.long	OF_DT_BEGIN_NODE
	.asciz	"PowerPC,970@1"
	.balign	4
	.long	OF_DT_PROP
	.long	0xc
	.long	0x3a
	.long	0x506f7765
	.long	0x7250432c
	.long	0x39373000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x3f
	.long	0x63707500
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x4b
	.long	0x1
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x4f
	.long	0x5f5e1000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x5f
	.long	0x1fca055
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x81
	.long	0x10000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x8e
	.long	0x8000
	.balign	4
	.long	OF_DT_END_NODE
	.long	OF_DT_END_NODE
	.long	OF_DT_BEGIN_NODE
	.asciz	"randomnode"
	.balign	4
	.long	OF_DT_PROP
	.long	0x13
	.long	0x9b
	.long	0xff007374
	.long	0x75666673
	.long	0x74756666
	.long	0x909090a
	.short	0xa0a
	.byte	0x0
	.balign	4
	.long	OF_DT_PROP
	.long	0x9
	.long	0xa2
	.long	0xa0b0c0d
	.long	0xdeeaadbe
	.byte	0xef
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0xa7
	.long	0x2
	.balign	4
	.long	OF_DT_PROP
	.long	0x14
	.long	0xab
	.long	0x61626300
	.long	0x12340000
	.long	0xa
	.long	0xb
	.long	0xc
	.balign	4
	.long	OF_DT_END_NODE
	.long	OF_DT_BEGIN_NODE
	.asciz	"memory@0"
	.balign	4
	.long	OF_DT_PROP
	.long	0x7
	.long	0x3f
	.long	0x6d656d6f
	.short	0x7279
	.byte	0x0
	.balign	4
	.globl	memreg
memreg:
	.long	OF_DT_PROP
	.long	0x10
	.long	0x4b
	.long	0x0
	.long	0x0
	.long	0x0
	.long	0x20000000
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0x2c
	.long	0x2
	.balign	4
	.long	OF_DT_END_NODE
	.long	OF_DT_BEGIN_NODE
	.asciz	"chosen"
	.balign	4
	.long	OF_DT_PROP
	.long	0xf
	.long	0xb1
	.long	0x726f6f74
	.long	0x3d2f6465
	.long	0x762f7364
	.short	0x6132
	.byte	0x0
	.balign	4
	.long	OF_DT_PROP
	.long	0x4
	.long	0xba
	.long	0x600
	.balign	4
	.long	OF_DT_END_NODE
	.long	OF_DT_END_NODE
	.long	OF_DT_END
	.globl	dt_struct_end
dt_struct_end:
_dt_struct_end:
	.globl	dt_strings_start
dt_strings_start:
_dt_strings_start:
	.asciz	"model"
	.asciz	"compatible"
	.asciz	"#address-cells"
	.asciz	"#size-cells"
	.asciz	"linux,phandle"
	.asciz	"name"
	.asciz	"device_type"
	.asciz	"reg"
	.asciz	"clock-frequency"
	.asciz	"timebase-frequency"
	.asciz	"linux,boot-cpu"
	.asciz	"i-cache-size"
	.asciz	"d-cache-size"
	.asciz	"string"
	.asciz	"blob"
	.asciz	"ref"
	.asciz	"mixed"
	.asciz	"bootargs"
	.asciz	"linux,platform"
	.globl	dt_strings_end
dt_strings_end:
_dt_strings_end:
	.globl	dt_blob_end
dt_blob_end:
_dt_blob_end:
