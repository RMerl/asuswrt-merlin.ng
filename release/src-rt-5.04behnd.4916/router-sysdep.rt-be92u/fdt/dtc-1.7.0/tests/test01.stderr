DTC: dts->asm  on file "test.dts"
Warning: "linux,boot-cpu" property is deprecated in blob version 2 or higher
Warning: /chosen has no "linux,stdout-path" property
Warning: /chosen has no "interrupt-controller" property
